apps/src/code-studio/hiddenStageRedux.js

```diff
@@ -34,42 +34,74 @@ const HiddenState = Immutable.Record({
   })
 });
 
+/**
+ * Validates that we never have multiple stagesBySection if we have STUDENT_SECTION_ID
+ * @throws If new state is invalid
+ */
+function validateSectionIds(state) {
+  if (state.getIn(['stagesBySection', STUDENT_SECTION_ID]) &&
+      state.get('stagesBySection').size > 1) {
+    throw new Error('Should never have STUDENT_SECTION_ID alongside other sectionIds');
+  }
+}
+
 /**
  * hidden stage reducer
  * Mapping of stage ids to bools indicating whether it's hidden or not
```

Comment:
this comment seems like it might be out of place or out of date -- please fix or remove.